tests/test_visitors/test_ast/test_classes/test_methods/data_model.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+import attr
+
+
+@attr.dataclass(frozen=True, slots=True)
+class TestMethodArgs(object):
+    """Test method arguments data wrapper."""
+
+    definition: str = attr.ib()
```

Comment:
We use `attrs.dataclass` with `auto_attribs=True`, so `= attr.ib()` is not required